tools/src/test/python/dlpx/virtualization/_internal/conftest.py

```diff
@@ -579,6 +588,7 @@ def basic_artifact_content(engine_api, virtual_source_definition,
         'rootSquashEnabled': True,
         'buildNumber': '2',
         'luaName': 'lua-toolkit-1',
+        'extendedStartStopHooks': False,
```

Comment:
Not your change at all but can you quickly check why this basic_artifact_content exist outside of the artifact_content fixture? seems odd to me that we need to have two fixtures for the same thing when it's pretty easy to parameterized them to have the information we'd want/need? (you don't need to fix anything if it might take long, but maybe file an issue to track cleaning this up if it is needed?)

Reply:
Hmm, I can't immediately find any reason for this to exist either.  It was added here http://reviews.delphix.com/r/48675/  before this was open-sourced. Neither the review nor the Jira ticket mentions anything about why this was required, instead of just using `artifact_content`.  I filed https://github.com/delphix/virtualization-sdk/issues/241 for this